### PR TITLE
API Assignment 3

### PR DESCRIPTION
**Pull request by waghmaresuyog**

### CODE REVIEW

File: API_Auto/src/test/java/testGetBrandList/BrandsTestCases.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import payload.Brand;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BrandsTestCases extends CrudOperation {
+
+    private static final Logger log = LogManager.getLogger("Brands.class");
```
Comment (anushreeSonak) [inline API_Auto/src/test/java/testGetBrandList/BrandsTestCases.java]: Please check your classname and give it accordingly

File: API_Auto/src/test/java/testGetBrandList/BrandsTestCases.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    @Test(priority = 1)
+    public void checkBrandStatusCode() {
+        Assert.assertEquals(getBrandsResponse().getStatusCode(), 200);
+        log.info("Status code is " + getBrandsResponse().getStatusCode());
+    }
+
+    @Test(priority = 3)
```
Comment (anushreeSonak) [inline API_Auto/src/test/java/testGetBrandList/BrandsTestCases.java]: Give sequence wise priority

File: API_Auto/src/test/java/testGetBrandList/BrandsTestCases.java (hunk 1 above)
Comment (waghmaresuyog) [reply API_Auto/src/test/java/testGetBrandList/BrandsTestCases.java]: Done

File: API_Auto/src/test/java/testGetBrandList/BrandsTestCases.java (hunk 2 above)
Comment (waghmaresuyog) [reply API_Auto/src/test/java/testGetBrandList/BrandsTestCases.java]: Done

**Review by anushreeSonak — APPROVED** [review]

File: API_Auto/src/test/java/base/CrudOperation.java [hunk 3]
```diff
@@ -5,18 +5,25 @@
...
+    //Get Request hit and return the response for the Product
     public Response getResponse() {
         Response response = RestAssured.get(url).then().extract().response();
         return response;
     }
 
-    // Post request hit and return the response
+    // Post request hit and return the response for product
     public Response postRequest() {
         Response response = RestAssured.post(url).then().extract().response();
```
Comment (prashantkawa) [inline API_Auto/src/test/java/base/CrudOperation.java:20]: Try sending Payload in POST request

File: API_Auto/src/test/java/payload/Brand.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package payload;
+
+public class Brand {
+
+    public String brandId;
+    public String brandName;
```
Comment (prashantkawa) [inline API_Auto/src/test/java/payload/Brand.java]: Use private here

**Review by prashantkawa — CHANGES_REQUESTED** [review]

File: API_Auto/src/test/java/payload/Brand.java (hunk 4 above)
Comment (waghmaresuyog) [reply API_Auto/src/test/java/payload/Brand.java]: done

File: API_Auto/src/test/java/base/CrudOperation.java (hunk 3 above)
Comment (waghmaresuyog) [reply API_Auto/src/test/java/base/CrudOperation.java:20]: No body for this API

**Review by prashantkawa — APPROVED** [review]

**Pull request merged**